publish.rb

```diff
@@ -31,7 +31,7 @@
     FileUtils.cp_r "_site/.", tmp
     Dir.chdir tmp
     system "git init"
-
+    
```

Comment:
I add the whitespace because it makes it much easier to copy-and-paste to those blank lines, especially when I include a newline in my copied code.

Whatever the case, I'll remove it -- I don't have strong feelings either way and see no reason to fight about inconsequential whitespace. Braces, however, are a different story... :wink: